modules/actionagent/callback.go

```diff
@@ -80,18 +85,30 @@ func (agent *Agent) SetCallbackReporter() {
 		agent.AppendError(err)
 		return
 	}
+	var fileStreamTimeoutSec uint64 = 60
```

Comment:
parse env at prepare step

Reply:
also put into easy use

Reply:
still parse env at `SetCallbackReporter`, and put into agent eay use